openstates/tx/votes.py

```diff
@@ -258,55 +242,71 @@ def clean_bill_id(bill_id):
     '[@class = "textpara"]',
     '[contains(translate(., "YEAS", "yeas"), "yeas")]',
 ]
-def record_votes(root, session):
+
+
+def record_votes(root, session, chamber):
     for el in root.xpath('//div{}'.format(''.join(vote_selectors))):
         mv = MaybeVote(el)
         if not mv.is_valid:
             continue
 
-        v = Vote(None, None, 'passage' if mv.passed else 'other', mv.passed,
-                 mv.yeas or 0, mv.nays or 0, mv.present or 0)
-        v['bill_id'] = mv.bill_id
-        v['bill_chamber'] = mv.chamber
-        v['is_amendment'] = mv.is_amendment
```

Comment:
I didn't find a new place to record `is_amendment` or `method` ('record' or 'viva voce') anymore.